x-pack/plugin/watcher/src/main/java/org/elasticsearch/xpack/watcher/common/http/HttpClient.java

```diff
@@ -285,6 +347,24 @@ public void close() throws IOException {
         public String getMethod() {
             return methodName;
         }
+
     }
 
+    private boolean isBlacklisted(String host) {
```

Comment:
nit: can we change this to `isWhiteListed`

Reply:
I made it so that you do not have to call `if (isWhiteListed() == false)` in the interceptors above - as this seemed more readable to me... happy to switch though

Reply:
I think keeping everything in terms of whitelist reads better.  _(I had a brief moment of where is the blacklist defined ?)_.  This is of course an opinion and in general will defer to the author's (your) opinion for these non-consequential decisions.

Reply:
true. switched to `isWhiteListed()`